client/blocks/login/social.jsx

```diff
@@ -126,12 +126,6 @@ class SocialLoginForm extends Component {
 				</p>
 			);
 		}
-
-		return (
-			<p className="login__social-text">
-				{ this.props.translate( 'Or log in with your existing social profile:' ) }
-			</p>
-		);
```

Comment:
I'm not quite sure how to test the flow where `this.props.linkingSocialService` is enabled. Any guidance would be appreciated.

Reply:
Have an account for `yurname@gmail.com` on WPCOM.
Try connect with google account for the same email, you'll be in the linking social flow

Reply:
Thanks for the explanation Yury!